cmake/FindSQLite3.cmake

```diff
@@ -61,9 +89,18 @@ find_package_handle_standard_args(SQLITE3
 
 mark_as_advanced(SQLITE3_LIBRARY SQLITE3_INCLUDE_DIR SQLITE3_LIBRARIES)
 
-if(NOT SQLite3_FIND_QUIETLY)
+if(NOT SQLITE3_FIND_QUIETLY)
```

Comment:
why this change? `SQLite3_FIND_QUIETLY` is correct variable name for the  `QUIET` option of `SQLite3` package.

Reply:
This was a search/replace typo that slipped in.  Fixed, thanks for spotting it.